boxoffice/static/js/views/item_collection.js

```diff
@@ -108,14 +109,12 @@ export const ItemCollectionView = {
         components: {TableComponent: TableComponent, AggChartComponent: AggChartComponent}
       });
 
-      main_ractive.on('navigate', function(event, method){
-        // kill interval
-        eventBus.trigger('navigate', event.context.url);
-      });
+      NProgress.done();
+
+      SideBarView.render('Dashboard', remoteData.org, config.id);
```

Comment:
Pass `remoteData.org` and `config.id` as part of a single object.
```javascript
      SidebarView.render('Dashboard', {'org_id': remoteData.org, 'ic_id': config.id}
````

This way, it becomes easier to add new items to the sidebar. The function should fail if you try to render the sidebar without the necessary arguments.